jupyterhub_moss/spawner.py

```diff
@@ -19,20 +26,18 @@
     for name in ("option_form.css", "option_form.js")
 }
 
-# Required resources per partition
-REQUIRED_RESOURCES_COUNTS = "max_nprocs", "max_mem", "gpu", "max_ngpus", "max_runtime"
 
 with open(local_path("batch_script.sh")) as f:
     BATCH_SCRIPT = f.read()
 
 try:
     BATCHSPAWNER_VERSION = importlib.metadata.version("batchspawner")
 except importlib.metadata.PackageNotFoundError:
-    BATCHSPAWNER_VERSION = None
+    BATCHSPAWNER_VERSION = ""
```

Comment:
Why is it needed to change this ?

Reply:
It's for type consistency, else it needs:
```python
try:
    BATCHSPAWNER_VERSION: Optional[str] = importlib.metadata.version("batchspawner")
except:
```
But actually, I think the try/except blocks should be removed, if `batchspawner` or `jupyterhub` are not available.. it's unlikely any of this works ;)